src/engraving/libmscore/breath.h

```diff
@@ -79,6 +79,8 @@ class Breath final : public EngravingItem
     EngravingItem* nextSegmentElement() override;
     EngravingItem* prevSegmentElement() override;
     QString accessibleInfo() const override;
+    void added() override;
+    void removed() override;
```

Comment:
Better to move these methods to the private part of the class

Reply:
done